frankfurt-eks-vpc.tf

```diff
@@ -0,0 +1,114 @@
+#
```

Comment:
This file name should have been with `frankfurt`

Reply:
renamed file to eks-vpc.tf